### PR TITLE
[VideoToolbox] Add support for Xcode13 beta5.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/VideoToolbox/VTVideoEncoder.cs [hunk 1]
```diff
@@ -113,6 +113,26 @@ public class VTVideoEncoder {
 #endif
 		public bool SupportsFrameReordering { get; private set; }
 
+#if !NET
+		[NoiOS, NoTV, MacCatalyst (15,0), NoMac, NoWatch]
+#else
+		[UnsupportedOSPlatform ("ios")]
+		[UnsupportedOSPlatform ("tvos")]
+		[SupportedOSPlatform ("maccatalyst1515.0")]
```
Comment (mandel-macaque) [inline src/VideoToolbox/VTVideoEncoder.cs:121]:
```suggestion
		[SupportedOSPlatform ("maccatalyst15.0")]
```

File: src/videotoolbox.cs [hunk 2]
```diff
@@ -214,6 +214,22 @@ interface VTCompressionPropertyKey {
...
 		[Field ("kVTVideoEncoderSpecification_EnableLowLatencyRateControl")]
 		NSString EnableLowLatencyRateControl { get; }
+
+		[TV (15, 0), Mac (12, 0), iOS (15, 0), MacCatalyst (15,0)]
+		[Field ("kVTCompressionPropertyKey_BaseLayerBitRateFraction")]
+		NSString BaseLayerBitRateFraction { get; }
+
+		[TV (15, 0), Mac (12, 0), iOS (15, 0), MacCatalyst (15,0)]
+		[Field ("kVTCompressionPropertyKey_EnableLTR")]
+		NSString  EnableLtr { get; }
```
Comment (rolfbjarne) [inline src/videotoolbox.cs:224]: Extra space:

```suggestion
		NSString EnableLtr { get; }
```

File: src/videotoolbox.cs [hunk 3]
```diff
@@ -608,6 +632,22 @@ interface VTEncodeFrameOptionKey {
 
 		[Field ("kVTEncodeFrameOptionKey_ForceKeyFrame")]
 		NSString ForceKeyFrame { get; } 
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("kVTEncodeFrameOptionKey_AcknowledgedLTRTokens")]
+		NSString AcknowledgedLTRTokens { get; }
```
Comment (rolfbjarne) [inline src/videotoolbox.cs:638]: `LTR` was lower-cased to `Ltr` further above:

```suggestion
		NSString AcknowledgedLtrTokens { get; }
```

File: src/videotoolbox.cs [hunk 4]
```diff
@@ -608,6 +632,22 @@ interface VTEncodeFrameOptionKey {
...
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("kVTEncodeFrameOptionKey_BaseFrameQP")]
+		NSString BaseFrameQP { get; }
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("kVTEncodeFrameOptionKey_ForceLTRRefresh")]
+		NSString ForceLTRRefresh { get; }
```
Comment (rolfbjarne) [inline src/videotoolbox.cs:646]:
```suggestion
		NSString ForceLtrRefresh { get; }
```

File: src/videotoolbox.cs [hunk 5]
```diff
@@ -608,6 +632,22 @@ interface VTEncodeFrameOptionKey {
...
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("kVTSampleAttachmentKey_RequireLTRAcknowledgementToken")]
+		NSString RequireLTRAcknowledgementToken { get; }
```
Comment (rolfbjarne) [inline src/videotoolbox.cs:650]:
```suggestion
		NSString RequireLtrAcknowledgementToken { get; }
```

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by spouliot — CHANGES_REQUESTED** [review]
forgot to submit review yesterday :(
largely identical to Rolf's comments

**Review by spouliot — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Unrelated failures: https://github.com/xamarin/maccore/issues/2409

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**